Firmware/ultralcd.cpp

```diff
@@ -5477,7 +5465,7 @@ static void lcd_tune_menu()
         MENU_ITEM_FUNCTION_P(_T(MSG_FILAMENTCHANGE), lcd_colorprint_change);
 #endif
     if (print_job_timer.isPaused()) {// Don't allow rehome if actively printing. Maaaaybe it could work to insert on the fly, seems too risky.
-        MENU_ITEM_FUNCTION_P(_T(MSG_AUTO_HOME), lcd_rehome_xy);
+        MENU_ITEM_GCODE_P(_T(MSG_AUTO_HOME),PSTR("G28 XY"));
```

Comment:
> Do home directly, G28 X Y resets MBL

If this is still true I think we should keep that comment to explain why the menu isnt shown when printing. 

Isnt this also bad if the print is paused?

Reply:
In case the user has to move the extruder or bed when paused for some maintenance, the Auto home of XY is the only way to be able to resume the print.
This is only for the worst case scenario when you have/had to move the extruder or bed for some reason.

Reply:
Approved. Losing the MBL is not good but it wont ruin the print if its not the first layer.